src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -6136,13 +6165,11 @@ void CalcHeatBalanceOutsideSurf(EnergyPlusData &state,
                     int RoughSurf = state.dataMaterial->Material(state.dataConstruction->Construct(ConstrNum).LayerPoint(1)).Roughness;
                     // Thermal absoptance of the exterior surface
                     Real64 AbsThermSurf = state.dataMaterial->Material(state.dataConstruction->Construct(ConstrNum).LayerPoint(1)).AbsorpThermal;
-
+                    HMovInsul = state.dataSurface->SurfMovInsulHExt(SurfNum);
                     // Check for outside movable insulation
-                    if (Surface(SurfNum).MaterialMovInsulExt > 0) {
-                        EvalOutsideMovableInsulation(state, SurfNum, HMovInsul, RoughSurf, AbsThermSurf);
-                        if (HMovInsul > 0)
-                            AbsThermSurf = state.dataMaterial->Material(Surface(SurfNum).MaterialMovInsulExt)
-                                               .AbsorpThermal; // Movable outside insulation present
+                    if (state.dataSurface->SurfMovInsulHExt(SurfNum) > 0) {
```

Comment:
Another place where arrays of surface roughness and absthermal could eliminate this if and repeated lookups from `dataConstruction`.